uint8 SUCCESS = 0
uint8 FAILURE_NOOBJECT = 1

int32 robot_id
geometry_msgs/Pose goal
---
int32 result